lib/Commands/BuildAllCommand.php

```diff
@@ -97,7 +97,10 @@ private function runCommand(string $command, array $arguments): int
     {
         $input = new ArrayInput(array_merge(['command' => $command], $arguments));
 
-        return $this->getApplication()->find($command)
-            ->run($input, new ConsoleOutput());
+        $command = $this->getApplication()?->find($command);
+
+        assert($command instanceof Command);
+
+        return $command->run($input, new ConsoleOutput());
```

Comment:
If `find()` returns, it will always return a command. The assertion looks a bit more specific as it needs to be. The critical part is the `getApplication()` call which might return `null`, so let's only cover that particular case.

```suggestion
        $application = $this->getApplication();

        assert($application !== null);

        return $application->find($command)->run($input, new ConsoleOutput());
```